README.md

```diff
@@ -82,6 +82,39 @@ The Hamler 0.1 compiler was forked from [PureScript][PureScriptHamler] 0.13.6. T
   make install
   ```
 
+**Build from source code(linux/macOS)**
+
+0. Install [asdf-vm](https://asdf-vm.com/#/core-manage-asdf-vm?id=install)
+
+1. Install asdf-vm [Erlang plugin](https://github.com/asdf-vm/asdf-erlang)
+
+  ```shell
+  asdf plugin-add erlang
+  ```
+
+2. Install Erlang
+
+  ```shell
+  asdf install
```

Comment:
I think clone hamler from the git repo should be executed before `asdf install`, because `asdf install` requires `.tool-versions`

Reply:
Oh, yes, you're right. I've updated the README accordingly.